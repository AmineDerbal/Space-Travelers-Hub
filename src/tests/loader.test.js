import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Loader from '../Components/Loader';

it('test loader render', () => {
  const { container } = render(<Loader />);
  const tree = renderer.create(<Loader />).toJSON();
  expect(container.firstChild.classList.contains('lds-ring')).toBe(true);
  expect(container.getElementsByClassName('lds-ring').length).toBe(1);
  expect(tree).toMatchSnapshot();
});
