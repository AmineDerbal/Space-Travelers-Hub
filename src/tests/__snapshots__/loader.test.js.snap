// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test loader render 1`] = `
<div
  className="lds-ring"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
